Editor/Scripts/DCCIntegration/DCCPluginDownloader.cs

```diff
@@ -82,8 +82,8 @@ void CopyDCCPluginsFromPackage(string dccPluginVersion, Action<string> onSuccess
 
             //Check several folders
             string[] folders = {
-                $"Library/PackageCache/{MESHSYNC_DCC_PLUGIN_PACKAGE}@{version}/Editor/Plugins~",                
-                $"Packages/{MESHSYNC_DCC_PLUGIN_PACKAGE}/Editor/Plugins~",
+                $"Library/PackageCache/{MESHSYNC_DCC_PLUGIN_PACKAGE}@{version}/Editor/Plugins",                
+                $"Packages/{MESHSYNC_DCC_PLUGIN_PACKAGE}/Editor/Plugins",
```

Comment:
Instead of removing paths with `~`, I think we should add paths with no `~` in addition to the existing paths, 
because it's better to avoid adding dependency of having to release a new MeshSyncDCCPlugins package.

Reply:
You mean adding a symlink from Plugins to Plugins~?

Reply:
Not really. 

I meant, we should keep MeshSync working with the latest MeshSyncDCCPlugins, which is `MeshSyncDCCPlugin@0.14.1-preview` at the moment, and for that, we need to add paths in the above variable instead of replacing the existing ones.

Reply:
I see it now, will do

Reply:
Done